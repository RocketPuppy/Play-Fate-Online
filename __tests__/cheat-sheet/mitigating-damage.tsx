import * as React from "react";
import * as renderer from "react-test-renderer";
import MitigatingDamage from "../../src/cheat-sheet/mitigating-damage";

test("renders all the pieces", () => {
  const render = renderer.create(<MitigatingDamage />);

  expect(render.toJSON()).toMatchSnapshot();
});
