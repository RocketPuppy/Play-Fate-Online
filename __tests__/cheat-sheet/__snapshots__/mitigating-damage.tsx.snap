// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders all the pieces 1`] = `
<div>
  <h2>
    Mitigating Damage
  </h2>
  <p>
    Fill in one stress box greater than or equal to the value of an attack, take one or more consequence, or fill in one stress box and take consequences—if you can’t do one of these three things, you’re taken out.
  </p>
</div>
`;
